operator/src/main/resources/scripts/utils_base.sh

```diff
@@ -234,6 +234,17 @@ function initAuxiliaryImage() {
     return
   fi
 
+  if [[ "$AUXILIARY_IMAGE_COMMAND" == "cp -R \$AUXILIARY_IMAGE_PATH/* \$AUXILIARY_IMAGE_TARGET_PATH" ]]; then
+    # Auxiliary image command is default, validate that AUXILIARY_IMAGE_PATH exists and is not empty.
+    if [ ! -d ${AUXILIARY_IMAGE_PATH} ]; then
```

Comment:
minor: `${AUXILIARY_IMAGE_PATH}` should ideally be `"${AUXILIARY_IMAGE_PATH}"` in case the path has spaces in it.  i'm sure we have forgotten to do this in many places, but might as well do it here regardless